expo/expo
# [dev-launcher] fix updates is not showing in the ext tab

# Why

a regression of #22866 where updates on the dev-launcher is not showing anymore.

# How

fix the logic

# Test Plan

1. on a blank sdk 49 project + `npx expo install expo-dev-client expo-updates`
2. publish an eas update
3. login in dev-launcher and check the extension tab

| Before | After |
| :----: | :----: |
| <img src="https://github.com/expo/expo/assets/46429/0961c67d-716c-4f39-ab43-8b909e62aded" width="50%"> | <img src="https://github.com/expo/expo/assets/46429/6660b734-84c7-4258-8981-837296fcfcd6" width="50%"> |

# Checklist
